activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,7 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
+    config.active_record.belongs_to_required_by_default = true
```

Comment:
I mean, if any more is used in an initializer they will have different configuration if the configurations are only applied in a after_initialize callback.